backend/Dockerfile

```diff
@@ -1,35 +1,20 @@
-# Pull official latest Python Docker image (Pulished with version 3.11.0)
-FROM --platform=linux/amd64 python:latest
+# Pull gclub/skywardai:0.0.1 base image (Base on Python 3.11.0-slim)
+FROM docker.io/gclub/skywardai:0.0.1
 
 # Set the working directory
-WORKDIR /usr/backend
-
-# Set up Python behaviour
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
-ENV VIRTUAL_ENV=/opt/venv
+WORKDIR /app
 
 # Set the server port
 EXPOSE 8000
 
-# Install system dependencies
-RUN apt-get update \
-  && apt-get -y install netcat-traditional gcc postgresql \
-  && apt-get clean
-
-# Install Python dependencies
-COPY ./requirements.txt ./
-RUN pip3 install -r requirements.txt
-
 # Copy all files
 COPY . .
 
-# Copy entrypoint.sh for auto connection with account_db service
-COPY ./entrypoint.sh .
-RUN chmod +x /usr/backend/entrypoint.sh
+# Add execution permissions to entrypoint.sh
+RUN chmod +x /app/entrypoint.sh
```

Comment:
As we discussed, could we leave a correct permission to the entrypoint.sh. So, we can copy it to container directly.

Reply:
Here is another request see https://github.com/SkywardAI/containers/issues/1